snowex_db/point_data.py

```diff
@@ -147,6 +153,9 @@ def _format_df(self):
             # Verify the sample column exists and rename to variable
             self._check_sample_columns()
 
+        if self._skip_format_df:
+            return
+
```

Comment:
Other performance improvement is to skip doing the below steps if we already done it once for the entire file.